doc/ci.md

```diff
@@ -2,60 +2,66 @@
 
 ## Triggers
 
-The pipeline will be executed for every Pull Request and every push to the repository.
+The pipeline will be executed ONLY for pushes to Pull Requests.
 
-If you want the pipeline to be executed again on a Pull Request, you can add a comment saying `jenkins build please`.
-This will most likely be the case if you suspect that something is wrong with the build server infrastructure.
+If you want the pipeline to be executed, you can add a comment saying `jenkins build please`.
 If problems persist, please create a new issue with the failing build log.
 
-For users with login credentials for Jenkins, you can manually execute the pipeline for a branch or pull request via the [Jenkins UI](https://build.libelektra.org).
-Please ask if you want to have login data.
+For users with login credentials for Jenkins, you can manually execute the pipeline for a branch or pull request via the [Jenkins UI](https://build.libelektra.org/job/PermaplanT/).
+Please ask if you need login data.
 
-## Cancel Concurrent Builds
+## Concurrent Builds
 
-Previous builds on the same branch will get aborted to save computing time on the nodes.
+Concurrent builds on the same branch are disabled.
 The master branch is excluded from this rule.
 
 ## Stages
 
-### Prerequisites (Schema)
+### Sanity stage
 
-Before we can actually execute checks or build the binaries, we need `schema.rs` and `definitions.ts`.
+This stage performs rapid checks like pre-commit, migrations and schema building.
 
-These can be automatically created with `./ci/build-scripts/build-schema.sh`.
+Local pre-commit currently does not perform the codespell hook, but it is performed in the pipeline.
+Cargo fmt, eslint and groovy linting is not performed through pre-commit.
 
-### Tests & Build
+### Tests and Build
 
-This is a parallel stage that has a 2 hour timeout and exits if one substage fails.
-It checkouts the current codebase and each stage runs inside a separate docker container.
+This is a parallel stage which fails fast (exits if one stage fails) or times out after 2 hours.
 
-Following substages exist:
+It can be subdivided in 3 categories:
 
-- Test and Build Mdbook
-- Test and Build Backend (With PostGIS sidecar container)
-- Test and Build Frontend
+- Parallel cargo stages
+- Sequential frontend stages
+- Sequential mdbook stages
 
-For more information about automated integration tests look [here](./tests/README.md).
+Following `hidden` checks are done:
 
-Every pipeline run has its own isolated database.
```

Comment:
Is this not true anymore?

Reply:
It is true, but I removed some information that I think is obvious or not important for the reader.